src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -13851,6 +13855,19 @@ INSERT INTO tab_container_parameters VALUES(5646,'ORGAN','VenousBlood_UmbilicalC
 INSERT INTO tab_container_parameters VALUES(5646,'ORGAN','VenousBlood_UmbilicalCord','Volume',1,0,1,1,'ORGAN_VOLUMES',9,'LOCAL','INDIVIDUAL',0);
 INSERT INTO tab_container_parameters VALUES(5646,'ORGAN','VenousBlood_UmbilicalCord','Weight (tissue)',1,1,0,0,'TISSUE_WEIGHT',1,'LOCAL','INDIVIDUAL',0);
 INSERT INTO tab_container_parameters VALUES(5647,'NEIGHBORHOOD','PlacentaMaternal_pls_PlacentaFetal_cell','Surface area (plasma/intracellular)',1,0,1,1,'SURFACE_AREA_PLS_INT',1,'LOCAL','INDIVIDUAL',0);
+INSERT INTO tab_container_parameters VALUES(1678,'COMPOUND','DRUG','Enable supersaturation',1,0,1,0,'COMPOUND_PARTICLE_DISSOLUTION',100,'PROPERTY','COMPOUND',1);

```

Comment:
`Enable supersaturation` is the parameter which must be added by the project conversion 7.4=>7.5